static/js/ajax-episode-search.js

```diff
@@ -224,8 +224,6 @@ $(document).ready(function() {
 
         $('.epManualSearch').on('click', function(event) {
             event.preventDefault();
-            var performSearch = '0';
-            var showAllResults = '0';
 
             // @TODO: Omg this disables all the manual snatch icons, when one is clicked
```

Comment:
It was implemented for a reason. Not sure if it was a good reason. If you want to revert this. We change it back.